index.html

```diff
@@ -0,0 +1,676 @@
+
+
+
+                <div class="rating-css">
+
+                    <h1>11. Rating Component</h1>
+                    <div class="star-icon">
+                        <input type="radio" name="rating1" id="rating1">
+                        <label for="rating1" class="fa fa-star"></label>
+                        <input type="radio" name="rating1" id="rating2">
+                        <label for="rating2" class="fa fa-star"></label>
+                        <input type="radio" name="rating1" id="rating3">
+                        <label for="rating3" class="fa fa-star"></label>
+                        <input type="radio" name="rating1" id="rating4">
+                        <label for="rating4" class="fa fa-star"></label>
+                        <input type="radio" name="rating1" id="rating5">
+                        <label for="rating5" class="fa fa-star"></label>
+                    </div>
+
+                    <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522star-icon%2522%253E%250A%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rating1%2522%2520id%253D%2522rating1%2522%253E%250A%2520%253Clabel%2520for%253D%2522rating1%2522%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Flabel%253E%250A%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rating1%2522%2520id%253D%2522rating2%2522%253E%250A%2520%253Clabel%2520for%253D%2522rating2%2522%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Flabel%253E%250A%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rating1%2522%2520id%253D%2522rating3%2522%253E%250A%2520%253Clabel%2520for%253D%2522rating3%2522%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Flabel%253E%250A%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rating1%2522%2520id%253D%2522rating4%2522%253E%250A%2520%253Clabel%2520for%253D%2522rating4%2522%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Flabel%253E%250A%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rating1%2522%2520id%253D%2522rating5%2522%253E%250A%2520%253Clabel%2520for%253D%2522rating5%2522%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+                        style=" display:block; width: 742px; height: 552px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+</iframe>
+                </div>
+
+
+
+            </div>
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+
+
+
+
+            <!-- MODAL COMPONENT -->
+
+            <div class="modal_header" id="modal_header">
```

Comment:
modal cancel button is overflowing container.
![image](https://user-images.githubusercontent.com/39916455/114027466-02d51480-9895-11eb-8b6d-7c6ab0ed49f6.png)

Reply:
When is it overflowing?? I can't find this issue??